defaults/main.yaml

```diff
@@ -12,6 +12,15 @@ clamav_epel_yumrepo_gpgkey: "{{ yumrepo_epel_gpgkey | default('https://dl.fedora
 # Additional EPEL YUM repo params
 clamav_epel_yumrepo_params: "{{ yumrepo_epel_params | default({}) }}"
 
+# In the event of network congestion
+clamav_network_max_retries: 0
```

Comment:
Not needed - see comments bellow.

Reply:
Same as above. It does not loop when clamav_network_max_retries is 0 (default).